scripts/sysman/engine.yml

```diff
@@ -60,6 +60,22 @@ members:
       name: "subdeviceId"
       desc: "[out] If onSubdevice is true, this gives the ID of the sub-device"
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Extension properties related Engine group properties"
```

Comment:
suggested reword "Extension properties related to Engine Groups" ?

Reply:
Fixed